mlir/lib/Dialect/Rock/IR/RockDialect.cpp

```diff
@@ -1514,6 +1514,33 @@ void WorkitemIdOp::inferResultRanges(ArrayRef<ConstantIntRanges> argRanges,
                                      SetIntRangeFn setResultRanges) {
   setResultRanges(getResult(), getIdRange("block_size", getOperation()));
 }
+
+//===-----------------------------------------------------===//
+// ReduceSumOp
+//===-----------------------------------------------------===//
+
+LogicalResult ReduceOp::verify() {
+  ReduceOp &op = *this;
+  auto axis = op.getAxis();
```

Comment:
Why `&op` instead of just calling the methods directly?

Also, can we get less `auto`?